common/openapi.yml

```diff
@@ -52,7 +52,10 @@ paths:
           $ref: '#/components/responses/ServerError'
     post:
       tags: [ datasets ]
-      description: Create a new dataset
+      description: |
```

Comment:
Did you mean to replace the `Create a new dataset` text here? Or maybe that text should go in a `summary` property instead of in `description`?

Reply:
The text is still in the description (see below), but I think for consistency we should switch these descriptions to the summary section instead.